var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -125,6 +133,18 @@ def install(self, spec, prefix):
         sh = which('sh')
         sh('./autogen.sh')
 
+        if self.get_arch() == 'ppc64le':
+            # update the config.sub/guess files
+            with working_dir("config"):
+                # get new config.guess and config.sub files
+                print 'Backing up existing config.[sub|guess] files\n'
+                subprocess.call("mv config.sub config.sub.orig", shell=True)
+                subprocess.call("mv config.guess config.guess.orig",
+                                shell=True)
+                print 'Downloading lastest config.[sub|guess] files\n'
+                subprocess.call("wget -O config.sub 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'", shell=True)
+                subprocess.call("wget -O config.guess 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'", shell=True)
```

Comment:
I subclassed Adios from AutotoolsPackage instead of Package--sorry, I was not aware of this before.